src/app/core/models/cards.ts

```diff
@@ -1,7 +1,19 @@
 export interface ICard {
+  type: string;
```

Comment:
type: 'rent' | 'sell';
Or you can create enum cardType and use it as a type here